File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2850.cs
using System;

class URI
{
    static void Main() {
        while(true) {
            string pal = Console.ReadLine();
            if (string.IsNullOrEmpty(pal)){
                break;
            }

            if(pal=="esquerda") Console.Write("ingles\n");
            else if(pal=="direita") Console.Write("frances\n");
            else if(pal=="nenhuma") Console.Write("portugues\n");
            else Console.Write("caiu\n");
        }
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2850.cpp
#include <iostream>

using namespace std;

int main() {
    string f;
    while(getline(cin, f)){
        if(f=="esquerda") cout << "ingles" << endl;
        else if(f=="direita") cout << "frances" << endl;
        else if(f=="nenhuma") cout << "portugues" << endl;
        else cout << "caiu" << endl;
    }

    return 0;
}